engine/crates/runtime/src/hooks.rs

```diff
@@ -39,6 +40,11 @@ impl<'a, T> Anything<'a> for T where T: serde::Serialize + serde::de::Deserializ
 pub type AuthorizationVerdict = Result<(), PartialGraphqlError>;
 pub type AuthorizationVerdicts = Result<Vec<AuthorizationVerdict>, PartialGraphqlError>;
 
+pub struct ChannelBuilder<'a, Sender: Clone> {
```

Comment:
Why Is `ChannelBuilder` here? It's a hooks wasi implementation detail, so should be in `runtime-local`